crates/types/src/consensus.rs

```diff
@@ -963,6 +966,25 @@ impl<TYPES: NodeType> Consensus<TYPES> {
         let epoch_number = epoch_from_block_number(next_block_number, self.epoch_height);
         Ok(TYPES::Epoch::new(epoch_number))
     }
+
+    /// Returns true if the `parent_leaf` formed an eQC for the previous epoch to the `proposed_leaf`
+    pub fn check_eqc(&self, proposed_leaf: &Leaf<TYPES>, parent_leaf: &Leaf<TYPES>) -> bool {
```

Comment:
This function is checking does `proposed_leaf` contain an eQC as it' justify QC, is that right?

Reply:
I was just making sure I was correct, if we already have the parent then it's fine to pass it in, but probably this function makes sense as just starting from the proposed_leaf